BaseFilter/sections/general_extensions.txt

```diff
@@ -272,6 +272,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/146786
+asianage.com#%#AG_onLoad(() => { try { document.body.innerHTML = document.body.innerHTML.replaceAll('ADVERTISEMENT', '')} catch {} })
```

Comment:
Could you please explain the necessity of using `try..catch`? Do you have an error in the console?

Reply:
Hm, ok. Could you also add `;` at the end of the rule?

Reply:
Done.